src/main/java/com/OrganizationController.java

```diff
@@ -17,17 +18,51 @@
 public class OrganizationController extends HttpServlet{
     private static final OrganizationService organizationService = new OrganizationService();
 
-        protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException,
-                IOException {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException,
+            IOException {
 
-            String name = request.getParameter("name");
-            Organization organization = organizationService.saveOrganization(name);
+//            Try 1
+//            StringBuffer jo = new StringBuffer();
+//            String line = null;
+//
+//            try {
+//                BufferedReader reader = request.getReader();
+//                while ((line = reader.readLine()) != null)
+//                    jo.append(line);
+//            } catch (IOException e) {
+//                e.printStackTrace();
+//            }
+//
+//            try {
+//                JsonObject jsonObject =  HTTP.toJSONObject(jo.toString());
+//            } catch (JSONException e) {
+//                // crash and burn
+//                throw new IOException("Error parsing JSON request string");
+//            }
+
+//            Try 2
+        StringBuilder org = new StringBuilder();
+        BufferedReader reader = request.getReader();
+        try {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                org.append(line).append('\n');
```

Comment:
```suggestion
                org.append(line);
```

I think we don't need the newline character because we don't print it anywhere.